### PR TITLE
New Adapter: Oraki

### PR DESCRIPTION
**Pull request by BenOraki**
## DOC
https://github.com/prebid/prebid.github.io/pull/5403

### CODE REVIEW

File: static/bidder-info/oraki.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+endpoint: "https://eu1.oraki.io/pserver"
```
Comment (onkarvhanumante) [inline static/bidder-info/oraki.yaml:1]: endpoint is not reachable

```
curl -i --location --request POST 'https://eu1.oraki.io/pserver'

HTTP/2 521
date: Fri, 28 Jun 2024 10:32:02 GMT
content-type: text/plain; charset=UTF-8
content-length: 15
report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=Vwp%2BVk0ag0PLQ%2FTPo7qSfCtFg2h41Qbo2UZqN2IhTZ3H21BX19f2g0GEM16nd35jJXMYrvs5Yb2SyeYdZR3bPLaiyLoQYI7ppJLMvO1s0J4wIxT8Ljo3MLRyLqSpWIk%3D"}],"group":"cf-nel","max_age":604800}
nel: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}
x-frame-options: SAMEORIGIN
referrer-policy: same-origin
cache-control: private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0
expires: Thu, 01 Jan 1970 00:00:01 GMT
server: cloudflare
cf-ray: 89ad023dcb94f3e1-BOM
alt-svc: h3=":443"; ma=86400

error code: 521
```

File: static/bidder-info/oraki.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+maintainer:
+  email: "ben@oraki.io"
```
Comment (onkarvhanumante) [inline static/bidder-info/oraki.yaml:3]: `ben@oraki.io` seems associated to an individual. Prebid recommends to use an group or organization email as maintainer email.

File: static/bidder-info/oraki.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://sync.oraki.io/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```
Comment (onkarvhanumante) [inline static/bidder-info/oraki.yaml:18]: redirect url is not working

```
https://sync.oraki.io/pbserver?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid={{.GPPSID}}&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Doraki%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%5BUID%5D%22
```
`sync.oraki.io` web server is down 

image

**Comment by przemkaczmarek** [comment]
URL still doesnt work
image

**Comment by onkarvhanumante** [comment]
Closing PR as this is in stale state for a month.  @BenOraki  can reopen this PR with updated changes

**Comment by BenOraki** [comment]
@onkarvhanumante @przemkaczmarek I have fixed the issues mentioned in this conversation. Can this pr be reopened or I need to create a new PR?

**Pull request closed without merging**